process/tsc_faq.md

```diff
@@ -66,7 +75,7 @@ For public meetings, LFX PCC provides a public link for sharing the meeting with
 
 ### Agenda
 
-The TSC Chairperson should share an agenda at least 24 hours prior to the meeting, if possible, with an ask to share additional agenda items. Projects can share via the public discussion lists or using a pull request on GitHub.
+The TSC Chairperson should share an agenda at least 24 hours before the meeting with an ask to share additional agenda items. Projects can share via the public discussion lists or a pull request on GitHub.
```

Comment:
"Projects can share" -> "Projects can share the agenda".
Also, should we add "via the public discussion lists, posting to the ASWF Slack, or a pull request on GitHub"?

Reply:
Cleaned this up to be more agnostic